### PR TITLE
[DO NOT MERGE] Spring 6 Upgrade testing

### PR DESCRIPTION
**Pull request by msnwatson**
## What was the problem?

Associated tickets or Slack threads:
- #?

## How to test this PR
- Step 1
- Step 2[^secrel]

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

File: svc-bie-kafka/build.gradle [hunk 1]
```diff
@@ -26,10 +26,9 @@ dependencies {
   implementation project(":shared:lib-bie-kafka")
 
   // avro schema and kafka
-  implementation 'org.apache.avro:avro:1.11.3'
+  implementation 'org.apache.avro:avro:1.11.2'
   implementation 'io.confluent:kafka-avro-serializer:7.4.0'
-  implementation 'org.apache.kafka:connect-api:3.6.0'
-  implementation 'org.apache.commons:commons-compress:1.24.0'
+  implementation 'org.apache.kafka:connect-api:3.5.1'
```
Comment (chengjie8) [inline svc-bie-kafka/build.gradle:31]: any reason we downgrade these versions?

File: svc-bie-kafka/src/main/java/gov/va/vro/services/bie/service/kafka/KafkaConsumer.java [hunk 2]
```diff
@@ -75,8 +73,6 @@ private BieMessagePayload handleGenericRecord(ConsumerRecord<String, Object> rec
         .diagnosticTypeCode((String) messageValue.get(KEY_DIAGNOSTIC_TYPE_CODE))
```
Comment (chengjie8) [inline svc-bie-kafka/src/main/java/gov/va/vro/services/bie/service/kafka/KafkaConsumer.java:73]: can we add the `ActionName` and `ActionResultName` changes back?

File: svc-lighthouse-api/build.gradle [hunk 3]
```diff
@@ -13,7 +13,7 @@ dependencies {
   // Provides endpoint for healthcheck
   implementation 'org.springframework.boot:spring-boot-starter-actuator'
 
-  def hapi_fhir_version="6.8.3"
+  def hapi_fhir_version="6.8.2"
```
Comment (chengjie8) [inline svc-lighthouse-api/build.gradle:16]: Also not sure why this is downgraded.

File: app/src/main/java/gov/va/vro/config/SecurityConfig.java [hunk 4]
```diff
@@ -66,24 +72,40 @@ public SecurityFilterChain apikeyFilterChain(HttpSecurity httpSecurity) throws E
...
+//                      claimInfo,
+//                      claimMetrics,
+//                      evidencePdf,
+//                      fullHealth,
+//                      healthAssessment,
+//                      immediatePdf,
+//                      ACTUATOR_URLS,
+//                      V3_URLS)
+//                  .authenticated();
+//            })
```
Comment (chengjie8) [inline app/src/main/java/gov/va/vro/config/SecurityConfig.java:103]: If the code block isn't necessary, remove it.

**Comment by msnwatson** [comment]
@chengjie8  just wanted to call out that this PR was just a quick branch off for some experiments

**Pull request closed without merging**